embulk-core/src/main/java/org/embulk/exec/BulkLoader.java

```diff
@@ -512,7 +512,7 @@ public void doCleanup(ConfigSource config, ResumeState resume)
     private ExecutorPlugin newExecutorPlugin(BulkLoaderTask task)
     {
         return Exec.newPlugin(ExecutorPlugin.class,
-                task.getExecConfig().get(PluginType.class, "type", new PluginType("local")));
+                task.getExecConfig().get(PluginType.class, "type", PluginType.LOCAL));
```

Comment:
Minor: Up to you. I'm not sure, we don't have major reasons so much to declare new `LOCAL` static field in `PluginType` class? `DefaultPluginType.create("local")` method call is good enough here instead?